app/services/proofing/lexis_nexis/date_formatter.rb

```diff
@@ -4,7 +4,8 @@ class DateFormatter
       attr_reader :date
 
       def initialize(date_string)
-        @date = parse_date_string(date_string)
+        # Can switch to Date.parse after next deploy
+        @date = DateParser.parse_legacy(date_string)
```

Comment:
@zachmargolis Should we revisit this re: "next deploy" comment?

(Noticed while auditing similar code)